### ISSUE TITLE
Update all examples to v0.8

### ISSUE THREAD

**Issue by PatrickAlphaC** [smartcontractkit/documentation#196]
- Chainlink API Call
- Chainlink Price Feeds
- Chainlink VRF
- Chainlink Keepers

**Comment by dmitrydao** [comment]
1. What is the proper path for main VRF contracts in v0.8? I can see it only in the dev directory. So do we use `@chainlink/contracts/src/v0.8/dev/VRFConsumerBase.sol`?
2. Do we use 0.8 Solidity or some specific version, like 0.8.4?

**Comment by dmitrydao** [comment]
Also, when I use the dev `ChainlinkClient`, I get the following issue:
<img width="1382" alt="Screenshot 2021-07-05 at 20 37 06" src="https://user-images.githubusercontent.com/5336968/124472663-f8adae00-ddd0-11eb-8f08-b07df2bdd434.png">

**Comment by PatrickAlphaC** [comment]
Yeah, we have to wait for it to move to master, and then we can approve this PR. Thanks for making it.

**Comment by dmitrydao** [comment]
Update. Most of the code samples work but there are two files missing in 0.8 version: `Owned.sol` and `Ownable.sol`. Also for some reason `FlagsInterface.sol` can't be found in the Remix. Here's the list of affected code samples:
1) Most of the data data providers, VRFD20.sol: `import "@chainlink/contracts/src/v0.6/Owned.sol”;` 
2) ATestnetConsumer.sol: import `”@chainlink/contracts/src/v0.6/vendor/Ownable.sol”;`
3) ArbitrumPriceConsumer.sol: `import "@chainlink/contracts/src/v0.8/interfaces/FlagsInterface.sol”;`

**Comment by PatrickAlphaC** [comment]
Can you make an issue on the Chainlink repo and link to this issue?

**Comment by PatrickAlphaC** [comment]
And ping @alexroan

**Comment by alexroan** [comment]
I mentioned in smartcontractkit/chainlink#5031, ownership contracts have been replaced.

To resolve this ticket we can use `ConfirmedOwner.sol` from the v0.8 directory.

**Comment by dmitrydao** [comment]
> Ownership contracts have [been replaced](smartcontractkit/chainlink#5031).
> 
> To resolve the Owner issue we can use `ConfirmedOwner.sol` from the v0.8 directory.
> 
> @dmitrydao
> 
> > Also for some reason FlagsInterface.sol can't be found in the Remix
> 
> Flags contracts for v0.8 are not yet released. See the [contracts changelog](https://github.com/smartcontractkit/chainlink/blob/develop/contracts/CHANGELOG.md#022---unreleased).

@alexroan 

Not sure why but `ConfirmedOwner.sol` can't be found in Remix.
<img width="1084" alt="Screenshot 2021-09-21 at 15 54 18" src="https://user-images.githubusercontent.com/5336968/134133213-8332fef8-fece-474c-9479-2de54158b798.png">

Here's the import that I'm using: `import "@chainlink/contracts/src/v0.8/ConfirmedOwner.sol";`
